mrp_unbuild_cust_alu_analytics/views/mrp_unbuild_views.xml

```diff
@@ -0,0 +1,95 @@
+<odoo>
+    <data>
+        <!-- Form View -->
+        <record id="mrp_unbuild_form_process_type_form_view" model="ir.ui.view">
+            <field name="name">mrp.unbuild.view.form (process_id)</field>
+            <field name="model">mrp.unbuild</field>
+            <field name="inherit_id" ref="mrp.mrp_unbuild_form_view"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='product_id']" position="before">
+                    <field name="process_type_id"/>
+                </xpath>
+            </field>
+        </record>
+
+        <!-- Tree View -->
+        <record id="mrp_unbuild_form_process_type_tree_view" model="ir.ui.view">
+            <field name="name">mrp.unbuild.view.tree (process_id)</field>
+            <field name="model">mrp.unbuild</field>
+            <field name="inherit_id" ref="mrp.mrp_unbuild_tree_view"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='product_id']" position="before">
+                    <field name="process_type_id" optional="0"/>
```

Comment:
Show or Hide?

Reply:
Hide

Reply:
OK, then let's use `hide` instead of `0`.

Reply:
Done